manage.py

```diff
@@ -15,23 +15,19 @@
 
 ADMIN_SQL_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'admin', 'sql')
 
-cli = click.Group()
+app = create_app()
 
 
-@cli.command()
+@app.cli.command()
 @click.option("--host", "-h", default="0.0.0.0", show_default=True)
 @click.option("--port", "-p", default=8080, show_default=True)
-@click.option("--debug", "-d", type=bool,
-              help="Turns debugging mode on or off. If specified, overrides "
-                   "'DEBUG' value in the config file.")
-def runserver(host, port, debug):
-    app = create_app(debug=debug)
+def runserver(host, port):
```

Comment:
flask comes with its own `run` command. Can we override that one here so that we don't have 2 with differing behaviour? (runserver has a different host/port and the extra_files arg, run doesn't)